dotnet-audit-logs-example/Views/Home/Index.cshtml

```diff
@@ -2,22 +2,118 @@
     ViewData["Title"] = "Send Events";
 }
 
-<body class="container_login">
-    <div class='flex_column'>
-        <div class="flex heading_div">
-            <img src="~/images/workos-logo-with-text.png" alt="workos logo">
+<body class="container_success">
+        <div class="logged_in_div_left flex_column">
+            <div class="flex logged_in_div_left_title">
+                <div class="flex-column">
+                    <div class="flex space-between">
+                        <div><h2>Audit Logs</h2></div>
+                        <div><i icon-name="settings" class="stroke_width=1"></i>
+                        </div>
+                    </div>
+                    <div><p class="sub-heading">.NET Example App</p></div>
+                </div>
+            </div>
+            <hr style="height:0.5px;width:75%;">
+            <div class="flex flex_column success-buttons">
+                <a href="/logout">
+                    <div class="flex sidebar-button selected">
+                        <div><i icon-name="settings-2" class="stroke_width=1"></i></div>
+                        <div>
+                            <p>Organizations</p>
+                        </div>
+                    </div>
+                </a>
+                <div id="myBtn" class="flex sidebar-button">
+                    <div><i icon-name="edit" class="stroke_width=1"></i></div>
+                    <div>
+                        <p>Audit Logs</p>
+                    </div>
+                </div>
+            </div>
         </div>
-
-        <h2>DotNet Audit Logs Example App</h2>
-        <form method="POST" action="/set_org">
-            <div class='flex_column'>
-                <div>
-                    <input type="text" id="org" name="org" class="text-input">
+        <div class="logged_in_div_right">
+            <div class="logged_in_nav">
+                <div class="flex">
+                    <div>
+                        <a href="https://workos.com/docs" target="_blank"><button class='button nav-item'>Documentation</button></a>
+                    </div>
+                    <div>
+                        <a href="https://workos.com/docs/reference" target="_blank"><button class='button nav-item'>API
+                            Reference</button></a>
+                    </div>
+                    <div>
+                        <a href="https://workos.com/blog" target="_blank"><button
+                            class='button nav-item blog-nav-button'>Blog</button></a>
+                    </div>
+                    <div>
+                        <a href="https://workos.com/" target="_blank">
+                            <img class="nav-image" src="../../wwwroot/images/workos_new.png" alt="link to workos.com">
```

Comment:
You don't need to include the full path with dotnet, you can just reference the images in the images folder like this: src="~/images/workos_logo_new.png"